### PR TITLE
v1.0.1 for esmtools

### PR DESCRIPTION
**Pull request by bradyrx**
@conda-forge/help-python 

Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/esmtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
```
Comment (marcelotrevisani) [inline recipes/esmtools/meta.yaml]: Please remove those comments

File: recipes/esmtools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - numpy
+    - pandas
+    - xarray
+    - scipy
+    - xskillscore >=0.0.4
+    - tqdm
+    - climpred >=1.0.1
+    - cftime
+    - cartopy
+    - pip
```
Comment (marcelotrevisani) [inline recipes/esmtools/meta.yaml]: Do you really need ``pip`` to run this project?

File: recipes/esmtools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+test:
+  imports:
+    - esmtools 
+
+about:
+  home: http://github.com/bradyrx/esmtools
+  license: MIT
+  license_file: LICENSE.txt
```
Comment (marcelotrevisani) [inline recipes/esmtools/meta.yaml]: Could you please add ``license_family: MIT`` ?

Comment (marcelotrevisani) [reply recipes/esmtools/meta.yaml]:
```suggestion
  license_family: MIT
  license_file: LICENSE.txt
```

File: recipes/esmtools/meta.yaml (hunk 2 above)
Comment (marcelotrevisani) [reply recipes/esmtools/meta.yaml]: I think you can remove it
```suggestion
```

**Comment by bradyrx** [comment]
Thanks @marcelotrevisani. All of your suggestions were implemented.

**Pull request closed without merging**